wolfcrypt/src/port/arm/armv8-curve25519.c

```diff
@@ -0,0 +1,7191 @@
+/* armv8-curve25519
```

Comment:
I don't see this new .c file in an .am file for proper distribution.  Did I miss it?

Reply:
The C file is the same as the assembly file just inline assembly instead of ATT format.
Either the asm or the C file are included.

Reply:
I'm asking about distribution.  Where is this .c file included in an .am file for proper distribution?

Reply:
Oops! Now added!